packages/plugin-core/src/components/lookup/buttons.ts

```diff
@@ -96,13 +97,26 @@ const selectionToNoteProps = async (opts: {
   switch (selectionType) {
     case "selectionExtract": {
       if (!_.isUndefined(document)) {
+        const ws = getDWorkspace();
+        const leaveTrace =
+          DConfig.getProp(ws.config, "lookup").note.leaveTrace || false;
         const body = "\n" + document.getText(range).trim();
         note.body = body;
         // don't delete if original file is not in workspace
         if (!ext.workspaceService?.isPathInWorkspace(document.uri.fsPath)) {
           return note;
         }
-        await VSCodeUtils.deleteRange(document, range as vscode.Range);
+        if (leaveTrace) {
+          const editor = VSCodeUtils.getActiveTextEditor();
+          await editor?.edit((builder) => {
+            const link = note.fname;
+            if (!_.isUndefined(selection) && !selection.isEmpty) {
+              builder.replace(selection, `![[${link}]]`);
+            }
```

Comment:
here it would be nice to use `NoteUtils.createWikiLink` so that we can handle link aliases and vault prefixes too.
Check out how `CopyNoteLink` handles creating wikilinks to see an example usage